### ISSUE TITLE
[Bug]: Videos are too Small in all the levels on mobile screen

### ISSUE THREAD

**Issue by Raj100** [Vinay-Khanagavi/Cyberbear.github.io#143]
Solution Increase the responsiveness of overall website.

https://vinay-khanagavi.github.io/Cyberbear.github.io/index_levels/index-level3.html

<img width="1409" alt="Screenshot 2024-10-26 at 4 27 06 PM" src="https://github.com/user-attachments/assets/be572f9b-1b80-40e6-9da6-35ef5636acd6">

**Comment by Vinay-Khanagavi** [comment]
Yes, you can work on this. Ensure it is well-optimized for mobile visibility, but also make sure it does not affect the current design in web.  @Raj100

**Comment by B1N4RY-P4R45173** [comment]
@Raj100 start with level1

**Comment by Vinay-Khanagavi** [comment]
@Raj100 Please Pull the code and keep it updated
